packages/design-system/token-transformer/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "private": true,
+  "name": "@eternagame/design-system-token-transformer",
+  "description": "Utility library for converting JSON design tokens into various output formats",
+  "version": "1.0.0-dev",
+  "license": "SEE LICENSE IN LICENSE",
+  "scripts": {
+    "build": "vite build"
```

Comment:
I deleted the extra commands (build:watch in particular) because of the interaction with tokens build:watch command. But we can discuss whether there's a better way to handle it.

Reply:
If you tried to directly run `build:watch`, that will cause problems since Nx doesn't really allow for long-running task dependencies - that's why `nx-spawn` exists. This is definitely a bit of a weird case though, and we can discuss further